src/qt/bantablemodel.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2011-2013 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_QT_BANTABLEMODEL_H
+#define BITCOIN_QT_BANTABLEMODEL_H
+
+#include "main.h"
+#include "net.h"
+
+#include <QAbstractTableModel>
+#include <QStringList>
+
+class ClientModel;
+class BanTablePriv;
+
+QT_BEGIN_NAMESPACE
+class QTimer;
+QT_END_NAMESPACE
+
+struct CCombinedBan {
+    CSubNet subnet;
```

Comment:
CSubNet is in netbase.h, do we need net.h or main.h here?
Would a class CSubNet; be sufficient and include netbase.h just in the .cpp?